packages/blocks/src/page-block/edgeless/components/auto-complete/edgeless-auto-complete.ts

```diff
@@ -290,39 +295,59 @@ export class EdgelessAutoComplete extends WithDisposable(LitElement) {
     const id = surface.addElement('connector', {
       mode: ConnectorMode.Orthogonal,
       strokeWidth: 2,
-      stroke: this._current.strokeColor,
+      stroke: (<ShapeElement>this.current).strokeColor,
       source,
       target,
     });
     return surface.pickById(id) as ConnectorElement;
   }
 
-  private _createShape() {
-    return this.edgeless.surface.addElement(this._current.type, {
-      ...this._current.serialize(),
-      text: new Workspace.Y.Text(),
-    });
+  private async _createElement() {
```

Comment:
Guess here should be `createEdgelessElement`, since here you are creating both phasor elements and blocks 🤔.